packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache

```diff
@@ -14,7 +14,7 @@
     {{/isSignedIn}}
   {{/showSuccessMessage}}
   <section>
-    <div class="graphic {{graphicId}}">{{#t}}Success{{/t}}</div>
+    <div class="graphic {{graphicId}}" role="img" aria-label="{{#t}}Successfully authenticated{{/t}}">{{#t}}Success{{/t}}</div>
```

Comment:
I think we can remove the text content of these elements (e.g. `{{#t}}Success{{/t}}`). This was likely an older method of aiding screen readers with visibly hidden text, but now that we're moving to a better approach we can probably remove them.